script/release/DSL-scripts/01.KIE_pushReleaseBranches.sh

```diff
@@ -0,0 +1,81 @@
+# clone the build-bootstrap that contains the other build scripts
+if [ "$SOURCE" == "community-branch" ]; then
+
+   # clone droolsjbm-build-bootstrap branch from droolsjbpm
+   git clone git@github.com:droolsjbpm/droolsjbpm-build-bootstrap.git --branch $BASE_BRANCH
+
+   # clone rest of the repos
+   ./droolsjbpm-build-bootstrap/script/git-clone-others.sh --branch $BASE_BRANCH --depth 70
+   
+   # checkout to local release names
+   ./droolsjbpm-build-bootstrap/script/git-all.sh checkout -b $RELEASE_BRANCH $BASE_BRANCH
+   
+   # add new remote pointing to jboss-integration
+   ./droolsjbpm-build-bootstrap/script/git-add-remote-jboss-integration.sh
+
+fi
+
+if [ "$SOURCE" == "community-tag" ]; then
+
+   # clone droolsjbm-build-bootstrap branch from droolsjbpm
+   git clone git@github.com:droolsjbpm/droolsjbpm-build-bootstrap.git --branch $BASE_BRANCH
+
+   # clone rest of the repos
+   ./droolsjbpm-build-bootstrap/script/git-clone-others.sh --branch $BASE_BRANCH --depth 70
+   
+   # add new remote pointing to jboss-integration
+   ./droolsjbpm-build-bootstrap/script/git-add-remote-jboss-integration.sh
+   
+   # get the tags of community
+   ./droolsjbpm-build-bootstrap/script/git-all.sh fetch --tags origin
+   
+   # checkout to local release names
+   ./droolsjbpm-build-bootstrap/script/git-all.sh checkout -b $RELEASE_BRANCH $TAG
+
+fi
+   
+if [ "$SOURCE" == "production-tag" ]; then
```

Comment:
there are 3 possibilities for a base of a release:
- base is a community branch
- base is a prod tag
- base is a community tag
I encountered these three different possibilities.